docs/api/schemas.md

```diff
@@ -105,6 +119,22 @@ AuthorizationObject can represent a "set" of objects, such as: all workspaces in
 | ---------------- | ------- | -------- | ------------ | ----------- |
 | `[any property]` | boolean | false    |              |             |
 
+## codersdk.BuildInfoResponse
+
+```json
+{
+  "external_url": "string",
+  "version": "string"
+}
+```
+
+### Properties
+
+| Name           | Type   | Required | Restrictions | Description                                                                                                                                                                   |
+| -------------- | ------ | -------- | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `external_url` | string | false    |              | External url is a URL referencing the current Coder version. For production builds, this will link directly to a release. For development builds, this will link to a commit. |
```

Comment:
`s/url/URL`, I'm not overly conscious about which way we stylize it here, but we should be consistent. Go comment should remain as-is preferably.

Reply:
Fixed. Fortunately we can write JS functions :)